### PR TITLE
Improve messaging during cluster operations

### PR DESCRIPTION
**Pull request by ndeodhar**
Diff includes:
1. Currently, yb-ctl operations do not print anything on console which is a confusing user experience. Added some basic messaging for every operation.
2. Removed t-shirt printing message
3. Changed some inconsistencies in `yb-ctl status` output
4. Improved help messaging
Some parts of the diff are copied over from @bmatican 's PR.

Output now looks like this:
```
$ ./bin/yb-ctl destroy
Destroying cluster.

$ ./bin/yb-ctl create
Creating cluster.
Waiting for cluster to be ready.
Initializing cluster. This may take up to a minute.
.....................................................................................................................
----------------------------------------------------------------------------------------------------
| Node Count: 1 | Replication Factor: 1                                                            |
----------------------------------------------------------------------------------------------------
| JDBC                : postgresql://postgres@127.0.0.1:5433                                       |
| YSQL Shell          : ./bin/ysqlsh                                                               |
| YCQL Shell          : ./bin/cqlsh                                                                |
| YEDIS Shell         : ./bin/redis-cli -h 127.0.0.1 -p 6379                                       |
| Web UI              : http://127.0.0.1:7000/                                                     |
| Cluster Data        : /Users/yugabyte/yugabyte-data                                              |
----------------------------------------------------------------------------------------------------

For more info, please use: yb-ctl status

$ ./bin/yb-ctl stop
Stopping cluster.

$ ./bin/yb-ctl add_node
Adding node.
Waiting for cluster to be ready.
----------------------------------------------------------------------------------------------------
| Node 3: yb-tserver (pid 47577)                                                                   |
----------------------------------------------------------------------------------------------------
| JDBC                : postgresql://postgres@127.0.0.3:5433                                       |
| YSQL Shell          : ./bin/ysqlsh                                                               |
| YCQL Shell          : ./bin/cqlsh                                                                |
| YEDIS Shell         : ./bin/redis-cli -h 127.0.0.3 -p 6379                                       |
| data-dir[0]         : /Users/yugabyte/yugabyte-data/node-3/disk-1/yb-data                        |
| yb-tserver Logs     : /Users/yugabyte/yugabyte-data/node-3/disk-1/yb-data/tserver/logs           |
----------------------------------------------------------------------------------------------------

$ ./bin/yb-ctl stop_node 3
Stopping node tserver-3.

$ ./bin/yb-ctl status
----------------------------------------------------------------------------------------------------
| Node Count: 3 | Replication Factor: 1                                                            |
----------------------------------------------------------------------------------------------------
| JDBC                : postgresql://postgres@127.0.0.1:5433                                       |
| YSQL Shell          : ./bin/ysqlsh                                                               |
| YCQL Shell          : ./bin/cqlsh                                                                |
| YEDIS Shell         : ./bin/redis-cli -h 127.0.0.1 -p 6379                                       |
| Web UI              : http://127.0.0.1:7000/                                                     |
| Cluster Data        : /Users/yugabyte/yugabyte-data                                              |
----------------------------------------------------------------------------------------------------
----------------------------------------------------------------------------------------------------
| Node 1: yb-tserver (pid 47449), yb-master (pid 47446)                                            |
----------------------------------------------------------------------------------------------------
| JDBC                : postgresql://postgres@127.0.0.1:5433                                       |
| YSQL Shell          : ./bin/ysqlsh                                                               |
| YCQL Shell          : ./bin/cqlsh                                                                |
| YEDIS Shell         : ./bin/redis-cli -h 127.0.0.1 -p 6379                                       |
| data-dir[0]         : /Users/yugabyte/yugabyte-data/node-1/disk-1/yb-data                        |
| yb-tserver Logs     : /Users/yugabyte/yugabyte-data/node-1/disk-1/yb-data/tserver/logs           |
| yb-master Logs      : /Users/yugabyte/yugabyte-data/node-1/disk-1/yb-data/master/logs            |
----------------------------------------------------------------------------------------------------
----------------------------------------------------------------------------------------------------
| Node 2: yb-tserver (pid 47502)                                                                   |
----------------------------------------------------------------------------------------------------
| JDBC                : postgresql://postgres@127.0.0.2:5433                                       |
| YSQL Shell          : ./bin/ysqlsh                                                               |
| YCQL Shell          : ./bin/cqlsh                                                                |
| YEDIS Shell         : ./bin/redis-cli -h 127.0.0.2 -p 6379                                       |
| data-dir[0]         : /Users/yugabyte/yugabyte-data/node-2/disk-1/yb-data                        |
| yb-tserver Logs     : /Users/yugabyte/yugabyte-data/node-2/disk-1/yb-data/tserver/logs           |
----------------------------------------------------------------------------------------------------
----------------------------------------------------------------------------------------------------
| Node 3: yb-tserver (Stopped)                                                                     |
----------------------------------------------------------------------------------------------------
| JDBC                : postgresql://postgres@127.0.0.3:5433                                       |
| YSQL Shell          : ./bin/ysqlsh                                                               |
| YCQL Shell          : ./bin/cqlsh                                                                |
| YEDIS Shell         : ./bin/redis-cli -h 127.0.0.3 -p 6379                                       |
| data-dir[0]         : /Users/yugabyte/yugabyte-data/node-3/disk-1/yb-data                        |
| yb-tserver Logs     : /Users/yugabyte/yugabyte-data/node-3/disk-1/yb-data/tserver/logs           |
----------------------------------------------------------------------------------------------------

$ ./bin/yb-ctl --help
usage: yb-ctl [-h] [--binary_dir BINARY_DIR] [--data_dir DATA_DIR]
              [--replication_factor REPLICATION_FACTOR]
              [--num_shards_per_tserver NUM_SHARDS_PER_TSERVER]
              [--timeout TIMEOUT] [--verbose] [--install-if-needed]
              {create,start,stop,destroy,restart,wipe_restart,add_node,remove_node,start_node,stop_node,restart_node,status,setup_redis}
              ...

positional arguments:
  {create,start,stop,destroy,restart,wipe_restart,add_node,remove_node,start_node,stop_node,restart_node,status,setup_redis}
    create              Create a new cluster
    start               Create a new cluster, or start existing cluster if it
                        already exists.
    stop                Stop the cluster
    destroy             Destroy the cluster
    restart             Restart the cluster
    wipe_restart        Stop the cluster, wipe all data files and start the
                        cluster as before. Will lose all the flags though.
    add_node            Add a new node to the cluster
    remove_node         Stop the specified node in the cluster
    start_node          Start the specified node in the cluster
    stop_node           Stop the specified node in the cluster
    restart_node        Restart the specified node in the cluster
    status              Get cluster information
    setup_redis         Setup YugaByte to support Redis API
<snipped output>
```

### CODE REVIEW

File: bin/yb-ctl [hunk 1]
```diff
@@ -140,6 +143,16 @@ def retry_call_with_timeout(fn, timeout_sec=MAX_WAIT_SECONDS):
         time.sleep(SLEEP_TIME_IN_SEC)
 
 
+def wait_for_proc_report_progress(proc):
```
Comment (mbautin) [inline bin/yb-ctl:146]: Add a docstring to this function with a description of what it does.

File: bin/yb-ctl [hunk 2]
```diff
@@ -736,18 +749,18 @@ class ClusterControl:
...
+                ("stop", "Stop the cluster"),
+                ("destroy", "Destroy the cluster"),
+                ("restart", "Restart the cluster"),
+                ("wipe_restart", "Stop the cluster, wipe all data files and start the cluster as "
                                  "before. Will lose all the flags though."),
-                ("add_node", "Add a new node to the current cluster"),
-                ("remove_node", "Stop a particular node in the cluster."),
-                ("start_node", "Start a particular node with flags."),
+                ("add_node", "Add a new node to the cluster"),
+                ("remove_node", "Stop the specified node in the cluster"),
```
Comment (mbautin) [inline bin/yb-ctl:763]: It is a bit confusing that the command is called "remove" but the description says "stop". How is this different from stop_node?

**Review by mbautin — APPROVED** [review]

File: bin/yb-ctl (hunk 2 above)
Comment (ndeodhar) [reply bin/yb-ctl:763]: It's not different right now. Both just stop the node. Eventually, we should support removing a node from the cluster altogether or then drop this option.

**Comment by schoudhury** [comment]
Looks good!

**Pull request merged**